Sample/SwiftUI/UserInfoView.swift

```diff
@@ -0,0 +1,100 @@
+import SwiftUI
+import RInAppMessaging
+
+@available(iOS 13.0, *)
+struct UserInfoView: View {
+
+    @State private var userIDTextFieldText: String = ""
+    @State private var idTrackerTextFieldText: String = ""
+    @State private var accessTokenuserIDTextFieldText: String = ""
+    @State private var isEmptyTextFieldAlertPresented = false
+    @State private var isDuplicateTrackerAlertPresented = false
+    @State private var isSuccessAlertPresented = false
+
+    var body: some View {
+        VStack(alignment: .center) {
+            VStack(alignment: .leading) {
+                Text("USER ID:")
+                    .fontWeight(.bold)
+                    .foregroundColor(Color(.darkGray))
+                TextField("", text: $userIDTextFieldText, onCommit: {
+                    hideKeyboard()
+                })
+                .textFieldStyle(.roundedBorder)
+            }
+            .alert(isPresented: $isEmptyTextFieldAlertPresented) {
+                Alert(title: Text("Invalid input format"), message: Text("Fill least one field"))
```

Comment:
Please put all strings (like alert title and desc) into `.strings` file.
(You probably also need to copy content of `String+IAM` from SDK code into Sample app to use `"<strings-key>".localized`)